travis_install.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
nit: I think we should endeavor not to clutter the top level with files. I generally put files like this in something like `Tests/Scripts`. Additionally I might name it `install_dependencies.sh`?

Btw, this will help very much when building on evergreen! Thanks for splitting it out.